Dev-Tools/doppler/tool.md

```diff
@@ -4,12 +4,105 @@
 
 [Home](../../Readme.md) / [Dev Tools](../dev-tools.md) / [Doppler](tool.md)
 
+Welcome to the wonderful world of Doppler. Doppler is the SecretOps tool that we use. It holds onto all of our environment variables and allows easy access to them while still being secure. 
+
+As an analogy from Westin puts it this way. Before we used Google Drive which forced everyone to copy and paste all env files. We gave our mom (Google Drive) our secrets, but when her kid (CSG developers) wanted to use the secret with dad, she just shouted it out. There could have been another kid (anyone on the internet) that tried listening from the street and so far we have just gotten lucky that it hasn't happened yet. With our databases this means that they could have gotten all student info on campus
+
+```mermaid
+---
+title: Before Doppler
+---
+%%{init: {'theme':'dark'}}%%
+flowchart LR
+    M(Mom - Google Drive)
+    O(Other Kid - anyone else)
+    K(Kid - CSG developer)
+    D[(Dad - other software)]
+    M --> K <--> D
+    M -.-> O <--> D
+```
+
+Now with Doppler, when a kid wants to talk to Dad, they must first request the secret from mom. Mom covertly sends the secret to the Kid in a sealed letter riding atop a cat. The kid cannot open the letter but can give it to Dad to start talking to him. In this scenario the other kid would have to capture the cat and find a way to open the letter before being able to talk to dad.
+
+```mermaid
+---
+title: After Doppler
+---
+%%{init: {'theme':'dark'}}%%
+flowchart LR
+    M(Mom - Doppler)
+    O(Other Kid - anyone else)
+    K(Kid - CSG developer)
+    D[(Dad - other software)]
+    M -- cat w/ letter --> K <--> D
+    M -.-x O
+```
+
 ## Installation
 
-### Desktop
+### Windows
```

Comment:
I disagree with using these instructions. I think we should use the default Doppler install path.

Reply:
How did you install it? I had to use scoop.